src/styles/_catalog.scss

```diff
@@ -1,14 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1,200px);
+  gap: 48px;
```

Comment:
so then you could set in the parent of the flex some settings to center the elements.